### ISSUE TITLE
Any chance on implementing zoomIn()?

### ISSUE THREAD

**Issue by nikolakurtic** [makbn/java_leaflet#4]

**Comment by makbn** [comment]
Hi Nikola. Im going to refactor the whole project soon. I'll consider to add the zoom functionality.

**Comment by nikolakurtic** [comment]
Nice to hear that!

**Comment by makbn** [comment]
Hi @nikolakurtic,
I've implemented the zoom functionalities in v1.9.4. You can find them here https://github.com/makbn/java_leaflet/tree/dev/1.9.4

```
map.getControlLayer().setZoom(5);
map.getControlLayer().zoomIn(2);
map.getControlLayer().zoomOut(1);
```

You can access the rest of the control methods under `map.getControlLayer()` layer.
